### ISSUE TITLE
unpin jemalloc

### ISSUE THREAD

**Issue by thomashampson** [mantidproject/mantid#37474]
jemalloc has been pinned to avoid crashes on IDAaaS. The pin was added in https://github.com/mantidproject/mantid/pull/37405

**Comment by peterfpeterson** [comment]
From v5.2.0 to 5.3.0 they changed to subpackages. Rather than unpinning, I suggest this be changed to a minimum pin of >=5.3.0 to avoid other issues

**Comment by thomashampson** [comment]
I think we will have to wait until this issue is resolved before we can move onward from 5.2.0:
https://github.com/jemalloc/jemalloc/issues/2472#issuecomment-2154840020

**Comment by thomashampson** [comment]
Closing in favour of #40249
